aws_cloudtrail_rules/aws_resource_made_public.py

```diff
@@ -24,9 +24,12 @@ def rule(event):
     if not parameters or event.get(
             'errorCode') == 'AccessDenied' or event_name == "":
         return False
-
+    if event.get('errorCode') == 'AccessDenied':
+        return False
     # S3
-    if event_name == 'PutBucketPolicy':
+
+    if event['eventName'] == 'PutBucketPolicy':
+        # Don't alert if access is denied
```

Comment:
I don't think we need these changes. The check added on line 27 already happens on line 25, and `event_name` already contains the value of event['eventName'].